spec/inquisition/outputter/doc_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe Inquisition::Outputter::Doc do
+  let(:documentation) { described_class.new(nil) }
+
+  describe '#generate_doc' do
+    let(:current_time) { Time.now.strftime('%d_%m_%Y') }
+    let(:directory) { Rails.root.to_s + '/inquisition' }
```

Comment:
Can we use `Inquisition::Configuration.instance.output_path` instead of `Rails.root.to_s + '/inquisition'`?